doc/source/uml/lattice.txt

```diff
@@ -2,13 +2,13 @@
 abstract class ABCLattice {
   name: str
   type: str
-  base_vect: tuple of double[2..3]
-  size: tuple of double[2..3]
-  origin: tuple of double[2..3]
+  base_vect: float[3]
+  size: tuple of int[3]
```

Comment:
Thanks @itziakos. Current implementation (or docstrings) of ABCLattice and Lattice seem to have the size attribute as tuple of int[3], which is an unneccessary requirement. I'll change both of them to int[3], and the uml diagrams as well.